core/http-auth-aws/src/main/java/software/amazon/awssdk/http/auth/aws/AwsS3V4HttpSigner.java

```diff
@@ -33,6 +32,6 @@ public interface AwsS3V4HttpSigner extends HttpSigner<AwsCredentialsIdentity> {
      * @return AwsS3V4HttpSigner
      */
     static AwsS3V4HttpSigner create() {
-        return new DefaultAwsS3V4HttpSigner();
+        return null;
```

Comment:
All of the subinterfaces (query, event, s3) are set to return null, because I didn't want to remove the interfaces as a whole without discussing it first. 

1. We should remove the sub-interfaces, we don't have a use for theme anymore.
2. The properties in the sub-interfaces (such as `EXPIRATION_DURATION`) are moved to the main v4 signer interface